values.yaml

```diff
@@ -177,7 +177,7 @@ zabbixweb:
   # -- User of database
   POSTGRES_USER: zabbix
   # -- Password of database
-  POSTGRES_PASSWORD: zabbix_pwd
+  POSTGRES_PASSWORD: zabbix
```

Comment:
Hi @fmubaidien!

Thanks for your contributition.

The problem is the line 59 of values.yaml file.

The password must the same in parameters:

postgresql.auth.password 
zabbixserver.POSTGRES_PASSWORD
zabbixweb.POSTGRES_PASSWORD

In the case, you need change only the ``postgresql.auth.password``  parameter to ``zabbix_pwd``